### ISSUE TITLE
Manually flatten migrations

### ISSUE THREAD

**Issue by niconoe** [inbo/vespa-watch#54]
Due to large changes, the migration history is becoming a bit heavy/messy to my taste.

We could manually replace the whole migration folder with two migrations: one for the models in their current state, and one for the initial data (species, page fragments, ...).

Pros:

- Simpler, cleaner, more solid code (would probably also solve or completely avoid other issues such as #52 )

Cons:

- Needs to drop/recreate database on all machines (but not really a problem since we don't have a production database)
- Estimated workload: 1 hour.

When to do it:

- Either soon, so we have more stable code (but we may want to do it again before production, if we make more "big changes".
- Either as close as "beta version" as possible, so we enters this beta phase with a migration folder that's as clean as possible.
- Either the next time it hurts to write another manual migration on top of the rest.

I think I'd vote for the first option.

**Comment by bartaelterman** [comment]
I'm pro

**Comment by peterdesmet** [comment]
If it doesn't make for a more complicated job, I would opt for 2: close to beta.

**Comment by stijnvanhoey** [comment]
I agree, as we can easily redo the setup and have not yet anything in production, we do this close to beta version.

**Comment by bartaelterman** [comment]
Worked here!
